core/src/components/badge/badge.native.vars.scss

```diff
@@ -0,0 +1,11 @@
+@import "../../themes/functions.font";
+@import "./badge.common.vars.scss";
```

Comment:
This file isn't using any of the common vars so we can remove.

```suggestion
```

Reply:
Its not directly, but I changed md and ios partials to import this native vars instead, as it seemed to be more correct in terms of architecture